src/wallet/coinselection.cpp

```diff
@@ -331,7 +332,7 @@ void OutputGroup::Insert(const CInputCoin& output, int depth, bool from_me, size
     // coin itself; thus, this value is counted as the max, not the sum
     m_descendants = std::max(m_descendants, descendants);
 }
-
+// checks if eligible for spending using the coinelibibilityfilter
```

Comment:
Is this necessary? I think it's redundant since the name of the function is self explanatory.
`bool OutputGroup::EligibleForSpending`